libraries/provider_rack_user.rb

```diff
@@ -26,11 +27,21 @@ def whyrun_supported?
           end
         end
 
+        # default rack password to '*'
+        rack_password = '*'
```

Comment:
I feel like this should be a parameter on the resource. Especially if we're calling it a default (implies you can change the defaults).